src/error.rs

```diff
@@ -180,3 +182,9 @@ impl From<ignore::Error> for Error {
     }
   }
 }
+
+impl From<serde_json::Error> for Error {
```

Comment:
I think `serde_json::Error`s are general enough that I'd like to avoid adding this `From` impl.